lib/widgets/features/profile/components/edit_how_can_i_help.dart

```diff
@@ -92,9 +71,11 @@ class EditProfileHowCanIHelp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final topThreeExpertises = expertises.sublist(0, 3);
+    final topThreeExpertises = expertises.take(Limits.profileExpertiseMaxSize);
     final additionalExpertises =
-        (expertises.length > 3) ? expertises.sublist(3) : [];
+        (expertises.length > Limits.profileExpertiseMaxSize)
```

Comment:
This is what i was thinking that we could do here, too, since it's not clear to me we that actually will retain these values. It also makes it easier for us to sync between our current app and the new one if we don't add a new field.

Reply:
It is an easy fix, but it is awkward to manage since both fields would be dealing with a single array and depending on the position of the elements to classify the as either 'top' or 'additional'. A lot of questions popped up with this approach, which is why I preferred to leave the 'additional' fields unimplemented for now.

Reply:
Holger merged the PR i provided with the additional expertises fields. There was a lot of confusion about the requirements here though so leaving this un-implemented sounds good for now.